File: Modul 4. Loop/PRAK405/PRAK405-23110817120003-NatalieGraceKatiandagho.py
x, y = map(int,input().split())
total1 = 0
total2 = 0

for a in range(1, x + 1):
    perkalian = a * y
    total1 += perkalian
    if a > 1:
        for b in range(a, 0, -1):
            print(f"({b} * {y})", end="")
            if b > 1:
                print(" + ", end="")
        print(f" = {total1}")
    elif a == 1:
        print(f"({a} * {y}) = {perkalian}")

    total2 += total1

print(total2)

File: Modul 4. Loop/PRAK405/PRAK405-23110817120003-NatalieGraceKatiandagho.c
#include <stdio.h>

void main()
{
    int x;
    int y;
    int perkalian;
    int total1 = 0;
    int total2 = 0;
    int a;
    int b;

    scanf("%d %d", &x, &y);

    for (a = 1; a <= x; a++)
    {
        perkalian = a * y;
        total1 += perkalian;
        if (a > 1)
        {
            for (b = a; b >= 1; b--)
            {
                printf("(%d * %d)", b, y);
                if (b > 1)
                {
                    printf(" + ");
                }
            }
            printf(" = %d \n", total1);
        }
        else if (a == 1)
        {
            printf("(%d * %d) = %d \n", a, y, perkalian);
        }
        total2 += total1;
    }
    printf("%d", total2);
}
